Tests/Services/NfieldCapiInterviewersServiceTests.cs

```diff
@@ -0,0 +1,337 @@
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using Nfield.Infrastructure;
+using Nfield.Services.Implementation;
+using Xunit;
+using Nfield.Models;
+using Moq;
+using System.Net.Http;
+using System.Net;
+using Newtonsoft.Json;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Nfield.Services
+{
+    /// <summary>
+    /// Tests for <see cref="NfieldCapiInterviewersService"/>
+    /// </summary>
+    public class NfieldCapiInterviewersServiceTests : NfieldServiceTestsBase
+    {
+        const string CapiInterviewersEndpoint = "CapiInterviewers/";
+
+        #region AddAsync
+
+        [Fact]
+        public void TestAddAsync_ServerAcceptsInterviewer_ReturnsInterviewer()
+        {
+            var interviewer = new Interviewer { UserName = "User X" };
+            var mockedNfieldConnection = new Mock<INfieldConnectionClient>();
+            var mockedHttpClient = CreateHttpClientMock(mockedNfieldConnection);
+            var content = new StringContent(JsonConvert.SerializeObject(interviewer));
+            mockedHttpClient
+                .Setup(client => client.PostAsJsonAsync(new Uri(ServiceAddress, CapiInterviewersEndpoint), interviewer))
+                .Returns(CreateTask(HttpStatusCode.OK, content));
+
+            var target = new NfieldCapiInterviewersService();
+            target.InitializeNfieldConnection(mockedNfieldConnection.Object);
+
+            var actual = target.AddAsync(interviewer).Result;
+
+            Assert.Equal(interviewer.UserName, actual.UserName);
+        }
+
+        #endregion
+
+        #region RemoveAsync
+
+        [Fact]
+        public void TestRemoveAsync_InterviewerIsNull_ThrowsArgumentNullException()
+        {
+            var target = new NfieldCapiInterviewersService();
+            Assert.Throws<ArgumentNullException>(() => UnwrapAggregateException(target.RemoveAsync(null)));
+        }
+
+        [Fact]
+        public void TestRemoveAsync_ServerRemovedInterviewer_DoesNotThrow()
+        {
+            const string InterviewerId = "Interviewer X";
+            var interviewer = new Interviewer { InterviewerId = InterviewerId };
+            var mockedNfieldConnection = new Mock<INfieldConnectionClient>();
+            var mockedHttpClient = CreateHttpClientMock(mockedNfieldConnection);
+            mockedHttpClient
+                .Setup(client => client.DeleteAsync(new Uri(ServiceAddress, CapiInterviewersEndpoint + InterviewerId)))
```

Comment:
```suggestion
                .Setup(client => client.DeleteAsync(new Uri(ServiceAddress, $"{CapiInterviewersEndpoint}{InterviewerId}")))
```

Reply:
A clear sign of how I left old code, old code. :D Will change.